src/System.Diagnostics.StackTrace/src/System/Diagnostics/StackTraceSymbols.CoreCLR.cs

```diff
@@ -0,0 +1,244 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+extern alias SRE; 
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+using Path = SRE::System.IO.Path;  
+using BitConverter = SRE::System.BitConverter;  
+
+namespace System.Diagnostics
+{
+    public class StackTraceSymbols : IDisposable
```

Comment:
This API is private/internal between mscorlib and this assembly even though it is “public”. It isn’t exposed by the reference assembly so I didn’t think it needed any review.